contracts/periphery/oracle/src/testing.rs

```diff
@@ -17,11 +16,11 @@ fn decimal_overflow() {
         Uint256::from(price_cumulative_current.wrapping_sub(price_cumulative_last)),
         time_elapsed,
     );
+
     println!("{}", price_average.to_string());
-    println!(
-        "{}",
-        Uint128::from(price_average.mul(Uint256::from(amount))).to_string()
-    );
+
+    let res: Uint128 = price_average.mul(Uint256::from(amount)).try_into().unwrap();
```

Comment:
Why? Keep previous code

Reply:
because, the trait From is not implemented for Uint128

Reply:
I see. Don't understand why do we have these "prints" in test but ok. That's fine